libs/client/dashboard/src/lib/components/tags-management/tag-form/tag-form.component.ts

```diff
@@ -15,25 +15,37 @@ export class TagFormComponent implements OnInit {
 
     tagForm = new FormGroup({
         name: new FormControl('', [Validators.required]),
-        desc: new FormControl('', [Validators.required, Validators.maxLength(100)]),
+        desc: new FormControl('', [Validators.maxLength(100)]),
```

Comment:
nit: Only 100 feels pretty restrictive, and I think text limits are kind of a bad habit we have all over the codebase. We should bump this, and enforce length limitations on the view side, rather than the data side.